image.go

```diff
@@ -142,11 +144,6 @@ func (i *image) Unpack(ctx context.Context, snapshotterName string) error {
 		return err
 	}
 
-	layers, err = images.DecryptLayers(ctx, i.ContentStore(), layers, i.dcparameters)
```

Comment:
images.DecryptLayers() can now also be removed. I can do that... but shouldn't forget. It's dead now.